SurveyApp/Modules/Home/HomeViewController.swift

```diff
@@ -52,6 +52,7 @@ extension HomeViewController: HomeViewInput {
     func configure() {
         setUpLayout()
         setUpViews()
+        setIdentifier()
```

Comment:
I think it should be plural `setIdentifiers` 😁

Reply:
I updated the code. Thank you 🙇‍♀️